aspnetcore/grpc/configuration/sample/GrcpService/Startup2.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+
+namespace GrcpService
+{
+    public class Startup2
+    {
+        // This method gets called by the runtime. Use this method to add services to the container.
+        // For more information on how to configure your application, visit https://go.microsoft.com/fwlink/?LinkID=398940
+#region snippet
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddGrpc().AddServiceOptions<MyService>(options =>
+            {
+                options.ReceiveMaxMessageSize = 3 * 1024 * 1024;
+                options.SendMaxMessageSize    = 3 * 1024 * 1024;
```

Comment:
For didactic purpose, doesn't use same values for ReceiveMaxMessageSize and SendMaxMessageSize.

Reply:
@luizcarlosfaria  
Por favor, recomende valores apropriados.
Como é isso @mairaw?

Reply:
If change to 
```csharp
options.ReceiveMaxMessageSize = 2 * 1024 * 1024;  // 2 megabytes
options.SendMaxMessageSize = 5 * 1024 * 1024;     // 5 megabytes 
```
we address this point. 

Thank you @Rick-Anderson

Reply:
@luizcarlosfaria você pode verificar se eu consertei todos os problemas `@ JamesNK` identificados?

@luizcarlosfaria can you check that I've fixed all the problems `@JamesNK` identified?

Reply:
Hi @Rick-Anderson, sorry about my terrible english. 
You can write in english, it's not a problem for me, i can understand very well. Write and speak is always a problem, sorry about that.

I'm working on your request now.

Reply:
Ah sorry I was late to help with translations (was out on vacation) but I'm glad you guys figured out and got it done! 👏